bucket4j-hazelcast-all/bucket4j-hazelcast/src/main/java/io/github/bucket4j/grid/hazelcast/serialization/SerializationUtilities.java

```diff
@@ -0,0 +1,84 @@
+package io.github.bucket4j.grid.hazelcast.serialization;
+
+import com.hazelcast.internal.util.StringUtil;
+import com.hazelcast.nio.serialization.Serializer;
+
+import java.text.MessageFormat;
+import java.util.*;
+import java.util.function.Supplier;
+
+public class SerializationUtilities {
+    public static final String TYPE_ID_BASE_PROP_NAME = "Bucket4jSerializers_TypeId_Base";
```

Comment:
I would prefer to rename to "bucket4j.haszelcast.serializer.type_id_base"

Reply:
Gonna change it right now.